diff --git i/configure w/configure
index be5e0a5..99b2e14 100755
--- i/configure
+++ w/configure
@@ -24739,7 +24739,7 @@ if ${ac_cv_lib_ncurses_mvaddch+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lncurses  $LIBS"
+LIBS="-lncurses -ltinfo $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */

@@ -24770,7 +24770,7 @@ fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_ncurses_mvaddch" >&5
 $as_echo "$ac_cv_lib_ncurses_mvaddch" >&6; }
 if test "x$ac_cv_lib_ncurses_mvaddch" = xyes; then :
-  GUI_LINK_OPTS_TERM='-lncurses'
+  GUI_LINK_OPTS_TERM='-lncurses -ltinfo'
 fi

   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for mvaddch in -ltermlib" >&5
